package.json

```diff
@@ -42,7 +42,7 @@
   ],
   "dependencies": {
     "@jsforce/jsforce-node": "^3.5.1",
-    "@salesforce/core": "^8.8.2",
+    "@salesforce/core": "^8.8.3",
```

Comment:
Please bump the version of `@jsforce/jsforce-node` to ^3.6.5, and `@salesforce/core` to ^8.8.5